code/game/objects/items/cigs_lighters.dm

```diff
@@ -987,21 +965,4 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		s.start()
 		vapetime -= vapedelay
 
-	if((obj_flags & EMAGGED) && vapetime >= vapedelay)
```

Comment:
there is no reason for this to be removed, and its also an unlisted change in the changelog?
it requires an EMAG to access this, and emags are strictly restricted to traitors. Thus admin only

Reply:
The changelog states removes explosions from cigars which gib abnos, and traitors are a event we do occasionally on Skeld

Reply:
this isnt explosion code, it makes clouds of smoke at your location accasionally. Explosion code's earlier